treestats_paper.tex

```diff
@@ -458,14 +469,12 @@ \subsection*{Site statistics}
 However, since these situations depend on multiple mutations occurring at a single site,
 they are relatively rare in practice.
 
-Now we have defined a general, abstract, framework.
-Is it good for anything?
-Next we give some examples.
+Having defined a general framework, we may now give concrete examples of functions related to common summary statistics.
 
 \begin{example}[Nucleotide diversity] \label{ex:site_diversity}
-    Suppose we want to compute the mean density of nucleotide differences
-    in a group $S$ of $n$ sequences.
-    To do this,
+    The nucleotide diversity of a sample is the average number of differences between all haplotypes, which is
+    equivalent to the sum of heterozygosity at all positions.
+    To calculate this statistic,
```

Comment:
average *density* of differences?

Reply:
As defined, the statistic is based on the number.  The density would be the number per site.

Reply:
The definition of a site stat is [here](https://github.com/petrelharp/treestats_ms/blob/82e5f2a1fc35a2257aeafe625f845962fd854aa3/treestats_paper.tex#L429), which includes dividing by the length of the corresponding chunk of the genome, so it's at least intended to be defined as the density. Did I miss something?

Reply:
> The definition of a site stat is [here](https://github.com/petrelharp/treestats_ms/blob/82e5f2a1fc35a2257aeafe625f845962fd854aa3/treestats_paper.tex#L429), which includes dividing by the length of the corresponding chunk of the genome, so it's at least intended to be defined as the density. Did I miss something?

No, this is correct.  My bad.

Reply:
It may be redundant, but there are places where `f` is written as number and not as density.  For example, [here](https://github.com/petrelharp/treestats_ms/blob/master/treestats_paper.tex#L474).  That is where I am drawn to look for a definition.  To make those a density, we need a `/[b-a]` for all of them.  But, that makes things ugly.

Reply:
Well, the summary function `f` *always* returns a number, for each site. The normalization appears in the definition of the *statistic* that one gets *from* the summary function.  So, `b-a` should definately not appear in the definition of `f`.

Reply:
I agree.  My habit is just to prefer examples to more formal definitions when I'm trying to implement things.

Reply:
I think I know what I wanted to say here:  the function, `f`, is defined based on the number of pairwise diffs, not the density.  The latter is handled by `span_normalise` later, as you note.